tests/dialects/test_hw.py

```diff
@@ -25,3 +46,132 @@ def test_inner_sym_target():
     assert sub_target
     assert not sub_target.is_op_only()
     assert sub_target.is_field()
+
+
+@irdl_op_definition
+class ModuleOp(IRDLOperation):
+    name = "module"
+    region: Region = region_def()
```

Comment:
```suggestion
    region = region_def()
```

Don't feel obliged to change this, just easier to read. Similarly below.